src/gateway/kong-enterprise/plugin-ordering/get-started.md

```diff
@@ -130,6 +130,28 @@ ordering:
     deck sync
     ```
 
+{% endnavtab %}
+{% navtab Kong Manager UI %}
+
+1. In Kong Manager, open the **default** workspace.
+2. From the menu, open **Plugins**, then click **Install Plugin**.
+3. Find the **Rate Limiting** plugin, then click **Enable**.
+4. Apply the plugin as **Global**, which means the rate limiting applies to all requests, including every service and route in the workspace.
+5. Complete only the following fields with the following parameters.
+    1. config.minute: `5`
+    2. config.policy: `local`
+    3. config.limit_by: `ip`
+    
+    Besides the above fields, there may be others populated with default values. For this example, leave the rest of the fields as they are.
+6. Click **Install**.
+7. From the **Rate Limiting** plugin page, click the **Ordering** tab.
+8. Click **Add ordering**.
+9. For **Before access**, click **Add plugin**.
+10. Choose **Key Auth** from the **Plugin 1** dropdown menu.
+11. Click **Update**.
+
+The rate limiting plugin now limits the amount of requests against all services and routes in the default workspace *before* {{site.base_gateway}} requests authentication.
+
 {% endnavtab %}
```

Comment:
Have you tried surrounding the navtabs with an `{% if_version %}` tag? In this case, that would be surrounding lines 134-156. Not sure if it works, worth a try.

Reply:
@lena-larionova I tried adding the conditional rendering, but it wasn't working locally. Not sure if I did something wrong or if it's just not possible to conditionalize a tab.

Reply:
Looks like it's not possible :( we'll have to ask Michael or Fabian for support on that.

For now, you could just add a note in the KM tabs that says something like "Kong Manager support for dynamic plugin ordering is available starting in Kong Gateway 3.1"